pkg/nodeconfig/mount.go

```diff
@@ -3,7 +3,7 @@ package nodeconfig
 import (
 	"github.com/k8ssandra/cass-operator/pkg/reconciliation"
 	"github.com/k8ssandra/k8ssandra-operator/pkg/cassandra"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Unnecessary aliasing? Besides we usually use an alias composed of the last part before the version + the version itself, so: `corev1`.

Reply:
Good catch. I blame my IDE :)